lms/djangoapps/course_api/blocks/transformers/milestones.py

```diff
@@ -36,11 +37,12 @@ def collect(cls, block_structure):
     def transform_block_filters(self, usage_info, block_structure):
         if not settings.FEATURES.get('ENABLE_PROCTORED_EXAMS', False):
             return [block_structure.create_universal_filter()]
+        # this needs to change ^^^
 
-        def is_proctored_exam_for_user(block_key):
+        def has_milestones_for_user(block_key):
```

Comment:
Since this internal method has gotten bigger, can you move it out into a separate top-level method?